openapi.yaml

```diff
@@ -427,9 +450,9 @@ parameters:
   memberId:
     name: memberId
     in: path
-    description: unique string
+    description: unique integer
     required: true
-    type: string
+    type: integer
```

Comment:
You will also need to update `Definitions.MemberId` now that it is a number instead of a string